command/sidechain/registration/register_validator.go

```diff
@@ -0,0 +1,235 @@
+package registration
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/command/polybftsecrets"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+)
+
+var (
+	stakeManager         = contracts.ValidatorSetContract
+	stakeFn              = contractsapi.ChildValidatorSet.Abi.Methods["stake"]
+	newValidatorEventABI = contractsapi.ChildValidatorSet.Abi.Events["NewValidator"]
+	stakeEventABI        = contractsapi.ChildValidatorSet.Abi.Events["Staked"]
+)
+
+var params registerParams
+
+func GetCommand() *cobra.Command {
+	registerCmd := &cobra.Command{
+		Use:     "register-validator",
+		Short:   "Registers and stake an enlisted validator",
+		PreRunE: runPreRun,
+		RunE:    runCommand,
+	}
+
+	setFlags(registerCmd)
+
+	return registerCmd
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.accountDir,
+		polybftsecrets.DataPathFlag,
+		"",
+		polybftsecrets.DataPathFlagDesc,
+	)
+
+	cmd.Flags().StringVar(
+		&params.configPath,
+		polybftsecrets.ConfigFlag,
+		"",
+		polybftsecrets.ConfigFlagDesc,
+	)
+	cmd.Flags().StringVar(
+		&params.stake,
+		stakeFlag,
+		"",
+		"stake represents amount which is going to be staked by the new validator account",
+	)
+	cmd.Flags().Int64Var(
```

Comment:
Nitpick: consistent formatting :slightly_smiling_face: 
```suggestion
	)
	
	cmd.Flags().StringVar(
		&params.stake,
		stakeFlag,
		"",
		"stake represents amount which is going to be staked by the new validator account",
	)
	
	cmd.Flags().Int64Var(
```

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/98b6e5890ea7c06fda67307fd63fe1387ddc36cd